vermin/source_visitor.py

```diff
@@ -1470,7 +1515,15 @@ def __extract_fstring_value(self, node):  # pragma: no cover
         value.append("{" + kvs + "}")
         break
 
-      elif isinstance(n, ast.Index):
+      elif sys.version_info >= (3, 9) \
+           and isinstance(
+             n,
+             (ast.Constant, ast.Name, ast.Slice, ast.Tuple)  # a[0] or a[i] or a[0:1] or a[(0,1)]
+           ):
```

Comment:
I'm thinking whether this is good since we don't have `ast.Index` to ensure that this is an index anymore. I'm writing just because an index may be these four.
Since `ast.Constant`, `ast.Name`, and `ast.Tuple` have been processed before this, do you think it is better to leave the code like this or leave `ast.Slice` only?